.github/workflows/release-quickstart.yml

```diff
@@ -0,0 +1,133 @@
+          QUICKSTART_VERSION="$($(go env GOPATH)/bin/ziti-ci -q get-current-version)"
+          # drop the leading 'v', if any
+          QUICKSTART_VERSION=${QUICKSTART_VERSION#v}
+          if ! [[ "${QUICKSTART_VERSION}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            # fail the job because we could not obtain the current version from ziti-ci
+            echo "ERROR: QUICKSTART_VERSION=${QUICKSTART_VERSION} is not a semver"
+            exit 1
+          elif [[ "${GITHUB_REF}" =~ ^refs/heads/(release-v|main$) ]]; then
+            # Set output parameters for release branches
+            echo "DEBUG: QUICKSTART_VERSION=${QUICKSTART_VERSION}"
+            echo QUICKSTART_VERSION="${QUICKSTART_VERSION}" >> $GITHUB_OUTPUT
+          else
+            # Append short sha for non-Ziti-release refs to identify quickstart docker images
+            # shipped from release-next
+            QUICKSTART_VERSION="${QUICKSTART_VERSION}-$(git rev-parse --short ${GITHUB_SHA})"
+            echo "DEBUG: QUICKSTART_VERSION=${QUICKSTART_VERSION}"
+            echo QUICKSTART_VERSION="${QUICKSTART_VERSION}" >> $GITHUB_OUTPUT
+          fi
+
+          # configure the env var used by the quickstart's Dockerfile to download the correct version of ziti for the
+          # target architecture of each image build by trimming the hyphenated short sha suffix so that the preceding
+          # release version of the ziti executable is installed in the quickstart container image
+          ZITI_OVERRIDE_VERSION=${QUICKSTART_VERSION%-*}
+          echo "DEBUG: ZITI_OVERRIDE_VERSION=${ZITI_OVERRIDE_VERSION}"
+          echo ZITI_OVERRIDE_VERSION="${ZITI_OVERRIDE_VERSION}" >> $GITHUB_OUTPUT
+
+      # This container differs in that :latest is pushed from branch release-next in addition to main and hotfix
+      # branches like releasev1.2.x. See https://github.com/openziti/ziti/issues/898
+      - name: Configure Quickstart Container
+        env:
+          IMAGE_REPO: ${{ env.ZITI_QUICKSTART_IMAGE }}
+          IMAGE_TAG:  ${{ steps.get_version.outputs.QUICKSTART_VERSION }}
+        id: tagprep_qs
+        shell: bash
+        run: |
+          DOCKER_TAGS=""
+          DOCKER_TAGS="${IMAGE_REPO}:${IMAGE_TAG}"
+          DOCKER_TAGS+=",${IMAGE_REPO}:latest"
+          echo "DEBUG: DOCKER_TAGS=${DOCKER_TAGS}"
+          echo DOCKER_TAGS="${DOCKER_TAGS}" >> $GITHUB_OUTPUT
+
+      - name: Build & Push Multi-Platform Quickstart Container Image to Hub
+        uses: docker/build-push-action@v3
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          context: ${{ github.workspace }}/quickstart/docker/image
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ steps.tagprep_qs.outputs.DOCKER_TAGS }}
+          build-args: |
+            ZITI_VERSION_OVERRIDE=${{ steps.get_version.outputs.ZITI_VERSION_OVERRIDE }}
+          push: true
+
+      - name: Manage Quickstart Latest Tag
+        uses: julbme/gh-action-manage-tag@v1
```

Comment:
is this just making a tag? doesn't ziti-ci do this?

Reply:
This step creates or updates the tag to the commit SHA of the closed PR's target branch. `ziti-ci` does not have the ability to create or update arbitrary tags, no.